client/app/bundles/KudosApp/actions/modalActions.jsx

```diff
@@ -0,0 +1,11 @@
+import { actionTypes } from "../constants/appConstants";
+
+function modalSwitch(activateModal) {
+  return {
+    type: actionTypes.MODAL_SWITCH,
+    activateModal
+  }
+}
+
+export default modalSwitch;
```

Comment:
I'm a bit confused by all the different terms used...

I think what's happening is that this `MODAL_SWITCH` action handles toggling the modal to be shown or hidden based on the `activateModal` flag.

It may be more straightforward to have two actions (or at least two functions), one to `showModal` and one to `hideModal`. They can be handled by the same function on the reducer side, but I think it would make what's going on here clearer.